digiroad2-oracle/src/main/resources/db/migration/V0_15__add_floating_reason_column.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE "ROAD_ADDRESS"
+  ADD (
+    "FLOATING_REASON" NUMBER(5,0)
```

Comment:
To my understanding it is bad practice to put column name inside brackets because in some cases it makes column name case sensitive.

Reply:
I'm removing this column.